src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -1,5 +1,10 @@
 package seedu.address.commons.core;
 
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import seedu.address.logic.parser.Prefix;
 import seedu.address.model.person.Person;
```

Comment:
Dependency on `model` is already a problem, and we are supposed to fix that separately. Now we are adding a dependency on `logic`. Would that be a further problem?

Reply:
After another look, I think this should be fine, as we plan to move Messages to the `logic` package.